### ISSUE TITLE
IGDM broken

### ISSUE THREAD

**Issue by rohangupta3** [ifedapoolarewaju/igdm#376]
Opens to show no chats and I cannot search anyone.

**Comment by SimonVanacco** [comment]
Can you please specify what OS you are using?

**Comment by rohangupta3** [comment]
How do I find out? I think it's the latest one.
    On Friday, 29 June 2018, 21:03:52 BST, Simon Van Accoleyen <notifications@github.com> wrote:  
 
 
Can you please specify what OS you are using?

**Comment by SimonVanacco** [comment]
Are you using Windows or an Apple Computer? That's all i need to know ;)

**Comment by kylerchin** [comment]
Hi! Have essentially the same issue here, I'm on a variant of Ubuntu Linux Distro: Pop OS https://system76.com/pop
I'm having some issues where when I log in, my chats don't show up:
![image](https://user-images.githubusercontent.com/7539174/55464955-07350b80-55b1-11e9-9f57-8c7f79d331fc.png)
I don't think it's my connection to instagram, since I tried this out on several different connections
Thanks!

**Comment by ifedapoolarewaju** [comment]
@kylerchin can you please logout and login again? Did you try that already?

**Comment by kylerchin** [comment]
Yep! I logged out and logged back in several times. I've been experimenting with some things, usually, when logging in from my reset ip address, it should shoot me an email telling me a new login as occurred. This doesn't happen via igdm. I can search for users and attempt to send them messages, it only pulls up chat history sometimes, even though I just uninstalled and reinstalled Igdm a few min ago. It takes a VERY LONG TIME to send messages, and my inbox is still blank.

I had my friends ping me several times, still nothing pops up:
![image](https://user-images.githubusercontent.com/7539174/55469701-df4aa580-55ba-11e9-848f-af43504e2fa3.png)
